crawler/config/test.py

```diff
@@ -20,7 +20,10 @@
         "prefix": "TEST",
         "lab_id_default": "TE",
         "backups_folder": "tmp/backups/TEST",
-        "sftp_file_regex": f"^TEST_{CENTRE_REGEX_SFTP_FILE}",
+        "sftp_file_regex_unconsolidated_surveillance": f"^TEST_{CENTRE_REGEX_SFTP_FILE_HERON}",
+        "sftp_file_regex_consolidated_surveillance": r"^TE-\d+\.csv$",
+        "sftp_file_regex_consolidated_eagle": r"^TE\d+\.csv$",
```

Comment:
Maybe make this a little more distinguishable?

Reply:
Ok done